js/app/src/components/Form/Form.svelte

```diff
@@ -23,7 +23,7 @@
 		border: var(--block-border-width) solid var(--border-color-primary);
 		border-radius: var(--block-radius);
 		background: var(--border-color-primary);
-		overflow: hidden;
```

Comment:
Does `overflow-y` also work? Seems like it would be a more targeted fix if so

Reply:
Made the change